### PR TITLE
Rename resin-supervisor to balena-supervisor

### PR DESCRIPTION
**Pull request by klutchell**
Maintain compatibility with resin namespaces for now.

Change-type: patch
Signed-off-by: Kyle Harding <kyle@balena.io>

### CODE REVIEW

File: upgrade-2.x.sh [hunk 1]
```diff
@@ -156,9 +156,9 @@ function compare_device_state() {
 function stop_services() {
     # Stopping supervisor and related services
     log "Stopping supervisor and related services..."
-    systemctl stop update-resin-supervisor.timer > /dev/null 2>&1
-    systemctl stop resin-supervisor > /dev/null 2>&1
-    ${DOCKER_CMD} rm -f resin_supervisor > /dev/null 2>&1 || true
+    systemctl stop update-balena-supervisor.timer > /dev/null 2>&1 || systemctl stop update-resin-supervisor.timer > /dev/null 2>&1
+    systemctl stop balena-supervisor  > /dev/null 2>&1 || systemctl stop resin-supervisor > /dev/null 2>&1
```
Comment (klutchell) [inline upgrade-2.x.sh:160]: `errexit` is set so we can't allow an exit code 1 in here.

- `systemctl stop balena-supervisor resin-supervisor` will exit 1
- `systemctl stop balena-supervisor resin-supervisor || true` won't catch failures to stop the service

File: upgrade-2.x.sh [hunk 2]
```diff
@@ -257,8 +257,8 @@ function upgrade_supervisor() {
 
 function error_handler() {
     # If script fails (e.g. docker pull fails), restart the stopped services like the supervisor
-    systemctl start resin-supervisor
-    systemctl start update-resin-supervisor.timer
+    systemctl start balena-supervisor resin-supervisor || true
+    systemctl start update-balena-supervisor.timer update-resin-supervisor.timer || true
```
Comment (klutchell) [inline upgrade-2.x.sh:261]: `errexit` is set so we can't allow an exit code 1 before the last service is restarted.

**Review by xginn8 — COMMENTED** [review]
one small comment otherwise lgtm if it works!

**Comment by klutchell** [comment]
> one small comment otherwise lgtm if it works!

I see the manual running instructions on the README now. I'll give it a test.

**Review by xginn8 — APPROVED** [review]

**Pull request merged**